.github/workflows/cleanup-actions.yml

```diff
@@ -32,7 +32,7 @@ jobs:
   
   delete-runs:
     name: "Delete old workflow runs"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
You can keep this on latest, doesn't matter (we'll probably need to switch to Ubuntu 22.04 soon anyway)